.github/workflows/release.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: cos-tool-arm64
-          path: dist/cos-tool_linux_arm64/cos-tool
+          path: dist/cos-tool_linux_arm64_v8.0/cos-tool
```

Comment:
Where does `v8.0` come from?  Why is the `amd64` `v1` instead?

Reply:
I believe it's because of armv8=arm64, armv7=armhf
https://en.wikipedia.org/wiki/ARM_architecture_family